docs/resource-policy/quick-start.md

```diff
@@ -0,0 +1,75 @@
+# Quick-start
+
+The following describes the minimum number of steps to get started with NRI
+Resource Policy plugin.
+
+## Pre-requisites
+
+- containerd or cri-o container runtime installed and running, and also
+  NRI feature enabled.
+- kubelet installed on your nodes
+
+## Setup NRI Resource Policy Plugin
+
+First, compile the resource plugins and create deployment image.
+
+```
+make && make images
+```
+
+### Deploy Daemonset
+
+The build/ directory will contain the needed images and deployment
+files. Copy the plugin .yaml file and corresponding image file into
+the node and deploy it there.
+
+For example:
+
+```
+ls build/images
+   nri-resource-policy-balloons-deployment.yaml
+   nri-resource-policy-balloons-image-ed6fffe77071.tar
+   nri-resource-policy-topology-aware-deployment.yaml
+   nri-resource-policy-topology-aware-image-9797e8de7107.tar
+```
+
+Copy the nri-resource-policy-topology-aware-deployment.yaml and the
+latest tar file, that was generated by the `make images` command.
+
+Note that containerd must have NRI support enabled and NRI is currently only
+available in 1.7beta or later containerd release.
+
+This will create a fresh config file and backup the old one if it existed:
+
+```
+[ -f /etc/containerd/config.toml ] && cp /etc/containerd/config.toml.backup
+containerd config default > /etc/containerd/config.toml
+```
+
+Edit the `/etc/containerd/config.toml` file and set `plugins."io.containerd.nri.v1.nri"`
+option `disable = true` to `disable = false` and restart containerd.
+
+If you are running cri-o, the NRI enabling can be done like this:
+
+```
+mkdir -p /etc/crio/crio.conf.d
+cat > /etc/crio/crio.conf.d/10-enable-nri.conf <<EOF
+[crio.nri]
+enable_nri = true
+EOF
+```
+
+Then you can deploy for example the topology-aware policy.
+
+```
+kubectl apply -f nri-resource-policy-topology-aware-deployment.yaml
+```
```

Comment:
How about adding one more step to verify that the policy is running and in action?